client/sites/hooks/use-transfer-with-software-start-mutation.ts

```diff
@@ -0,0 +1,70 @@
+import { useMutation, UseMutationResult } from '@tanstack/react-query';
+import wpcom from 'calypso/lib/wp';
+
+type transferWithSoftwareResponse = {
```

Comment:
I think we should use PascalCase for types.
```suggestion
type TransferWithSoftwareResponse = {
```